kong/plugins/aws-lambda/schema.lua

```diff
@@ -1,3 +1,14 @@
+local function check_status(status)
```

Comment:
@thibaultcha lazy me, was actually thinking about it and found the hmac-auth schema validation spec earlier. 😅  Added in 292d9a2404a1a0e40104c6346f14b11cd8b73013.